### DISCUSSION TITLE
5.1 sound

### DISCUSSION THREAD

**Discussion by Darksimmsy** [mpv-player/mpv#15170] [Q&A] [answered]
Hi does MPV support 5.1 by default if plugged into a 5.1 sound system like a amp via HDMI?

**Comment by danbezerra** [comment, marked as answer]
Yes, just add the following line specifying what formats your amp supports

audio-spdif=ac3,dts,dts-hd,eac3,truehd 

If you use the one above, all formats, including TrueHD, DTA HD MA, DTS X,  and Atmos will be enabled.
